src/main/java/com/foundrysoftware/minecraft/plugins/PluginTemplate.java

```diff
@@ -26,12 +51,102 @@ public void onDisable(){
     @Override
     public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command, @NotNull String label, String[] args) {
         if (label.equalsIgnoreCase("simple")){
-            if (sender instanceof Player){
-                Player me = (Player)sender;
+            if (sender instanceof Player) {
+                Player me = (Player) sender;
                 me.sendMessage("Plugin Working");
                 return true;
             }
-        }
+                if (label.equalsIgnoreCase("hawkeye")){//check command label
+                    if(args[0].equalsIgnoreCase("tnt")){//check if the second word is tnt
+                        if(sender instanceof Player){
+                            Player me =(Player)sender;
+                            PlayerInventory inventory = me.getInventory();
+                            for(int i = 0; i<inventory.getSize(); i++){
+                                ItemStack item = inventory.getItem(i);
+                                        if(item == null){
+                                            continue;
+                                        }
+                            if(isArrow(item)==true){
+                                item.setType(Material.TIPPED_ARROW);
+                                item.setAmount(item.getAmount());
+                                PotionMeta potionMeta = (PotionMeta)item.getItemMeta();
+                                potionMeta.setBasePotionData(new PotionData(PotionType.JUMP));
+                                int duration =10;
+                                int amplifier =10;
+                                boolean Visible=true;
+                                potionMeta.addCustomEffect(new PotionEffect(PotionEffectType.CONDUIT_POWER,duration,amplifier),Visible);
+                                item.setItemMeta(potionMeta);
+
+                            }
+                            }
+
+                        }
+                    }else{
+                        if(sender instanceof Player){
```

Comment:
this looks like duplicate code but with a different arrow effect, so we should be "fixing" this by looking at what is different between to the two variants. Do the rest first and we will come back to this

Reply:
thats because tnt is custom while the other one is for already fitted effects